integration_tests/tests/consistency/consistency_ad_group_report.sql

```diff
@@ -0,0 +1,45 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+with prod as (
+    select
+        ad_group_id,
+        sum(clicks) as clicks, 
+        sum(impressions) as impressions,
+        sum(spend) as spend
```

Comment:
Can we include the conversion metrics in these tests (even if just commented out) so we don't forget them when we make future releases.

Reply:
Same comment for other consistency tests